notebooks/Notebook2.md

```diff
@@ -2,33 +2,59 @@
+I think the biggest decision I made based on the critique were syntax changes.
+Firstly, I separated the element and transition files since that seemed like there was a general consensus on that.
+Also, I realized that the transition syntax is regular which makes things easier, whereas the element syntax is not.
+To help with parsing that, based on Prof. Wiedermann's suggestion I decided to actually slightly modify my syntax to fit with YAML which has already built interpreters making it easy to parse.
+
+I also liked the suggestion of separating the elements and transitions by frame, so I made sure the syntax would allow that (although I don't enforce it).
+I also thought a bit about what defaults I will want to have since that does seem to be a pretty useful standard feature when designing presentations.
+I currently have some default styles in a sheet I will link to every output HTML (that can easily be overwritten), but in terms of parsing, it's a bit more complicated to make sure the right elements have the right default styles etc.
 
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
-important design decisions, changes to previous decisions, open questions,
-exciting milestones, preliminary results, etc. Feel free to include images
-(e.g., a sketch of the design or a screenshot of a running program), links to
-code, and any other resources that you think will help clearly convey your
-design process.
+This work I primarily worked on trying to interpret the element syntax I have into a preliminary HTML file.
+This also involved changing my syntax quite a bit as I made design decisions.
+Since I switched to YAML, it was pretty straightforward to parse the syntax into a usable form, but writing the HTML proved to be a bit more difficult.
+I tried several different libraries that each had their strengths and weaknesses, but eventually I settled on `lxml` since it allows straightforward creation of DOM elements, can parse HTML files and HTML text, and can easily select DOM elements based on selectors (with the help of `cssselect`).
+
+In terms of the actual logic of my interpreter, I tried to keep it pretty basic so it would allow for creating the basic case I made last week.
+I know I'll definitely have to change a bit to make it more flexible for different designs (especially in terms of layout), but I tried to make it codable later on without too much reorganizing of the parsing process.
 
 ## Questions
 
 **What is the most pressing issue for your project? What design decision do
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+Currently, I'm still very open to any feedback about the syntax.
+I changed the transition syntax a bit and I'm planning to make the interpreter for that this week, so I guess any ideas of what's good/what could be improved with that syntax would be most helpful.
```

Comment:
I think your "example_keyframes.frame" syntax looks fine except for where you listed the elements with dashes. I think the dashes make title-frame-box look like it's all one element that the transition is being applied to. Instead I think you should just put spaces between different elements.